### PR TITLE
Add OpenFoodFacts as hard

### PR DESCRIPTION
**Pull request by Yoanndp**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -17992,5 +17992,16 @@
         "domains": [
             "ciceksepeti.com"
         ]
+    },
+
+    {
+        "name": "OpenFoodFacts",
+        "url": "https://openfoodfacts.org",
+        "difficulty": "hard",
+        "notes": "In order to delete your OpenFoodFacts account, you need to fill out a <a href=\"https://blog.openfoodfacts.org/en/account-deletion\">form</a> and send an email.",
```
Comment (tupaschoal) [inline _data/sites.json:18001]:
```suggestion
        "url": "https://blog.openfoodfacts.org/en/account-deletion",
        "difficulty": "hard",
        "notes": "In order to delete your OpenFoodFacts account, you need to fill out the form and send an email.",
```

We use the URL to point to the most direct deletion link possible.
Question: do you really have to fill the form AND send an e-mail? Feels so bureaucratic

**Review by tupaschoal — COMMENTED** [review]
Will need to be moved within the file so it can continue to be sorted by name

File: _data/sites.json (hunk 1 above)
Comment (Yoanndp) [reply _data/sites.json:18001]: Hi, thanks for the feedback! I tried to delete my account via the form and never got a response. I contacted the email on their page (https://world.openfoodfacts.org/terms-of-use) claiming my right to erasure (GDPR), but I haven't heard back yet.

Comment (tupaschoal) [reply _data/sites.json]: How long ago did you fill the form? And when did you send the email? Just so we can establish a maximum wait time to actually describe it as impossible

Comment (Yoanndp) [reply _data/sites.json]: I filled out the form a month ago, and sent 2 emails : one 2 weeks ago, and one this Monday

Comment (tupaschoal) [reply _data/sites.json]: I suggest we change the difficulty to impossible then, and if you get a reply, we come back and change it back to hard.

File: _data/sites.json [hunk 2]
```diff
@@ -11364,6 +11364,17 @@
         ]
     },
 
+    {
+        "name": "OpenFoodFacts",
+        "url": "https://blog.openfoodfacts.org/en/account-deletion",
+        "difficulty": "hard",
+        "notes": "In order to delete your OpenFoodFacts account, you need to fill out the form and send an email.",
+        "email": "contact@openfoodfacts.org",
```
Comment (tupaschoal) [inline _data/sites.json:11372]:
```suggestion
        "difficulty": "impossible",
        "notes": "Though there is a form to delete your OpenFoodFacts account, it seems no one is tending to it.",
```

File: _data/sites.json (hunk 1 above)
Comment (Yoanndp) [reply _data/sites.json]: Alright, if everything is good on your end, you can merge. I'll let you know if I get an answer from them! Thanks for the collaboration.

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**